modules/statics/src/networks.ts

```diff
@@ -120,7 +120,7 @@ class Ada extends Mainnet implements AdaNetwork {
   utxolibName = 'cardano';
   poolDeposit = 500000000;
   stakeKeyDeposit = 2000000;
-  explorerUrl = 'https://explorer.cardano.org/en/';
+  explorerUrl = 'https://explorer.cardano.org/en/transaction?id=';
```

Comment:
This gonna is gonna be parsed by the UI as
`https://explorer.cardano.org/en/transaction + txId + id=`

Reply:
This should be fixed in that repo, not here as this is the base definition.

Reply:
Ah I see that now. Not sure how I'd work around this though aside from hard coding something in the UI